src/WGPUCanvas.jl

```diff
@@ -31,22 +31,24 @@ elseif Sys.iswindows()
     include("glfwWindows.jl")
 end
 
+struct ErrorCanvas <: AbstractWGPUCanvas end
 
-function WGPUCore.getCanvas(s::Symbol)
-    if s==:OFFSCREEN
-        return defaultCanvas(OffscreenCanvas)
+function WGPUCore.getCanvas(s::Symbol, size::Tuple{Int, Int} = (500,500))
```

Comment:
julia prefers `;` for kwargs i think.

Reply:
Is there a `getCanvas` function in `WGPUCore`

Reply:
I don't see getCanvas function with size argument. So I wonder if we need to override `WGPUCore` getCanvas.

Reply:
> julia prefers ; for kwargs i think.
While thats true, since we're exposing only one argument/option and not a bunch of them,
in this case, simple arguments (with defined type) make more sense.

As for `WGPUCore.getCanvas`, we're not sure but `WGPUCore.getCanvas(:OFFSCREEN)` will behave the same as `WGPUCore.getCanvas(:OFFSCREEN, (500,500))`, so it preserves the previous structure. Hence, everything will remain the same.
We've tested this in WGPUgfx.jl and the change and all examples work. You can look at that work in https://github.com/JuliaWGPU/WGPUgfx.jl/pull/30

Reply:
Yeah. kwargs comment makes sense. I usually don't care either, trying to raise bar for myself. On overriding part I wonder if its the case for 1.8, 1.9 etc... I remember fiddling with errors along those lines. Anyways. Hopefully this works on all recent julia versions.  

I tested examples too. They work on my side too (on Mac M1 aarch architecture).